float32[4] goal
builtin_interfaces/Duration duration
builtin_interfaces/Time t_start
---
bool success
string message